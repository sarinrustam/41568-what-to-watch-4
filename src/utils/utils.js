export const FILTER_ALL_GENRES = `All Genres`;

export const PAGE_FILTERS = [`Overview`, `Details`, `Reviews`];

export const COUNT_LIMIT_MOVIES = 8;

export const CommentLength = {
  MIN: 50,
  MAX: 400,
};

export const Buttons = {
  ESC: `ESC`,
};

export const extend = (a, b) => {
  return Object.assign({}, a, b);
};

export const AppRoute = {
  LOGIN: `/login`,
  ROOT: `/`,
};
