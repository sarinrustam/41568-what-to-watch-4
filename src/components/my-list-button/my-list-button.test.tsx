import * as React from "react";
import * as renderer from "react-test-renderer";
import {noop} from "../../utils/utils";
import MyListButton from "./my-list-button";

describe(`MLB test`, () => {
  it(`Should MLB render correctly`, () => {
    const tree = renderer.create(
        <MyListButton
          isFavorite={true}
          onToggleButton={noop}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
