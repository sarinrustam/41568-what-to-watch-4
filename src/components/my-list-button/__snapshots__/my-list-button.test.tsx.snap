// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MLB test Should MLB render correctly 1`] = `
<button
  className="btn btn--list movie-card__button"
  onClick={[Function]}
  type="button"
>
  <svg
    height="14"
    viewBox="0 0 18 14"
    width="18"
  >
    <use
      xlinkHref="#in-list"
    />
  </svg>
  <span>
    My list
  </span>
</button>
`;
